services/ui/serverless.yml

```diff
@@ -31,7 +31,7 @@ custom:
   cloudfrontDomainName: ${ssm:/configuration/${self:custom.stage}/cloudfront/domainName, ""}
   iamPath: ${ssm:/configuration/${self:custom.stage}/iam/path, ssm:/configuration/default/iam/path, "/"}
   iamPermissionsBoundaryPolicy: ${ssm:/configuration/${self:custom.stage}/iam/permissionsBoundaryPolicy, ssm:/configuration/default/iam/permissionsBoundaryPolicy, ""}
-  wafLoggingBucket: ${cf:ui-waflog-s3-bucket-${self:custom.stage}.WaflogsUploadBucketArn}
+  wafLoggingBucket: ${param:WaflogsUploadBucketArn, ""}
```

Comment:
this was throwing an error when we tried to fetch the output to get the branch url because the param is managed by serverless compose but our output script looks in the service directly. I added the empty string option to make the error go away. open to other suggestions